examples/embeddable_explorer/public/hello_world_embeddable_example.tsx

```diff
@@ -56,7 +62,7 @@ export function HelloWorldEmbeddableExample({ helloWorldEmbeddableFactory }: Pro
             `input` prop may be used to declaratively update current embeddable input
           </EuiText>
           <EuiPanel data-test-subj="helloWorldEmbeddablePanel" paddingSize="none" role="figure">
-            <EmbeddableRenderer embeddable={new HelloWorldEmbeddable({ id: 'hello' })} />
+            <EmbeddableRenderer embeddable={new helloWorldEmbeddable({ id: 'hello' })} />
```

Comment:
I would capitalize all the constructors so this is `new HelloWorldEmbeddable`.

Reply:
It's not so important as this is an example, but we might isolate components better to refactor this to render props pattern.